centreon/www/install/insertTopology.sql

```diff
@@ -175,6 +175,7 @@ INSERT INTO `topology` (`topology_name`, `topology_url`, `readonly`, `is_react`,
 
 -- Add unified view page entry
 INSERT INTO `topology` (`topology_name`, `topology_url`, `readonly`, `is_react`, `topology_parent`, `topology_page`, `topology_group`, `topology_order`) VALUES ('Resources Status', '/monitoring/resources', '1', '1', 2, 200, 1, 2);
+INSERT INTO `topology` (`topology_name`, `topology_url`, `readonly`, `is_react`, `topology_parent`, `topology_page`, `topology_group`, `topology_order`) VALUES ('Dashboard (beta)', '/home/dashboards', '1', '1', 1, 104, 1, 2);
```

Comment:
Missing `topology_feature_flag` field
```suggestion
INSERT INTO `topology` (`topology_name`, `topology_url`, `readonly`, `is_react`, `topology_parent`, `topology_page`, `topology_group`, `topology_order`, `topology_feature_flag`) VALUES ('Dashboard (beta)', '/home/dashboards', '1', '1', 1, 104, 1, 2, 'dashboard');
```